experimental/torchprep/torchprep/fusion.py

```diff
@@ -0,0 +1,22 @@
+#TODO: Add helpers with triton for optimized fused ops
+import torch
+from pathlib import Path
+from typing import Optional
+from .utils import load_model
+from .format import Device, materialize_tensors, parse_input_format
+
+
+def _fuse(model_path : Path, input_shape : Path, output_name : str = "fused_model.pt", device : Device = Device.cpu) -> Optional[torch.nn.Module]:
+    model = load_model(model_path, device)
+    input_tensors = materialize_tensors(parse_input_format(input_shape))
+
+    try:
+        model = torch.jit.trace(model,input_tensors)
+    except Exception as e:
+        print(f"{model_path} is not torchscriptable")
+        return
+
+    optimized_model = torch.jit.optimize_for_inference(model)
```

Comment:
"optimize_for_inference" is experimental, and the result varies largely between models and devices. It will be good to use this with caution/ some benchmarks/ let user decide on this choice.

Reply:
Yeah this tool is really to just make it easy for users to try out optimizations and profile them too easily, intent is not to curate too much